streaming/src/main/scala/org/apache/spark/streaming/dstream/FileInputDStream.scala

```diff
@@ -223,6 +266,11 @@ class FileInputDStream[K, V, F <: NewInputFormat[K,V]](
    */
   private def isNewFile(path: Path, currentTime: Long, modTimeIgnoreThreshold: Long): Boolean = {
     val pathStr = path.toString
+    // Reject file if it start with _
+    if (path.getName().startsWith("_")) {
```

Comment:
Could you add this filter to `FileInputDStream.defaultFilter`? 

@tdas is it a bug that `FileInputDStream.defaultFilter` only ignores files that start with `.`? Hadoop uses the following path filter (FileInputFormat.hiddenFileFilter):
```Java
    private static final PathFilter hiddenFileFilter = new PathFilter() {
        public boolean accept(Path p) {
            String name = p.getName();
            return !name.startsWith("_") && !name.startsWith(".");
        }
    };
```
Since SparkContext.textFile uses `TextInputFormat`, it uses this filter and ignores files that start with "-" or ".". Should Streaming be consistent with Core?